sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/LoggerProviderConfigurationTest.java

```diff
@@ -140,7 +138,8 @@ void configureBatchLogRecordProcessor() {
                 assertThat(worker)
                     .extracting("queue")
                     .isInstanceOfSatisfying(
-                        Queue.class, queue -> assertThat(JcTools.capacity(queue)).isEqualTo(2));
+                        ArrayBlockingQueue.class,
+                        queue -> assertThat(queue.remainingCapacity()).isEqualTo(2));
```

Comment:
BatchLogRecordProcessor uses ArrayBlockingQueue, so this test abstraction wasn't needed (probably a copy paste from similar test for BatchSpanProcessor:

https://github.com/open-telemetry/opentelemetry-java/blob/1e763b28f0b3266cbb3fd6ffbfd5a5bf5b622654/sdk/logs/src/main/java/io/opentelemetry/sdk/logs/export/BatchLogRecordProcessor.java#L82